shakenfist/db.py

```diff
@@ -238,6 +239,7 @@ def place_instance(instance_uuid, node):
         return
```

Comment:
Does this check block the recording of a placement attempt? If not, maybe it shouldn't be called?

Reply:
So, a new attempt shouldn't be allowed to use the current node, so it shouldn't? That said, there might be a bug in the caller then?